plugin/evm/orderbook/contract_events_processor.go

```diff
@@ -484,24 +499,56 @@ func (cep *ContractEventsProcessor) PushtoTraderFeed(events []*types.Log, blockS
 		timestamp, _ := args["timestamp"]
```

Comment:
`unnecessary assignment to the blank identifier (S1005)go-staticcheck`

Reply:
fixed